### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by Basir-Mohammadi**
**Here we:**
**-Add pop-up window functionality to make the site dynamic.**
**-Add object to JS file to make project function dynamic.**

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that involves JavaScript without ESLINT being set up, Kindly make sure your ESLinter linters is set up and all passing before submitting the PR.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -11,499 +11,259 @@
...
-          <ul class="lists">
-            <li>
-              <a href="#"> <h6>html</h6></a>
-            </li>
 
-            <li>
-              <a href="#"> <h6>Css</h6></a>
-            </li>
+      <section class="projects"></section>
+      <div class="grid-container project-cards" id="port"></div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindy make sure your popup window is visible currently when I click to `see project` button it's taking me to the Github profile page, kindly fix it and ensure the popup is displayed and its design matches the design given.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @basirking and @ShaheerCH , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Basir-Mohammadi) [reply index.html]: Hi @danmainah it's working properly from my side.!

Comment (Basir-Mohammadi) [reply index.html]: @danmainah All are same as the figma.

Comment (Basir-Mohammadi) [reply index.html]: @danmainah  line 94 should be deleted sorry I forgot to get rid of it, it is useless code.

Comment (Basir-Mohammadi) [reply index.html]: @danmainah please check once more.

Comment (Basir-Mohammadi) [reply index.html]: Hi, @topeogunleye  please check once more.

Comment (topeogunleye) [reply index.html]: Hi, @basirking, KIndly submit for review again if you have met all requirements

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @basirking 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**